CapstoneProject/Assets/Scripts/Core/Controller/Dialogue/DialogueManager.cs

```diff
@@ -87,37 +129,97 @@ public bool LoadScriptObject()
     {
         bool success = true;
 
+        //Loads the dialogue based on the current scene NPC
+        if (gameState.currentNPC == "Whirl")
+        {
+            MoveObjectToList(whirlDialogue);
+            displayPortraitObject.GetComponent<UnityEngine.UI.Image>().sprite = whirlPortraitImage;
+        }
+        else if (gameState.currentNPC == "Emelia")
+        {
+            MoveObjectToList(emeliaDialogue);
+            displayPortraitObject.GetComponent<UnityEngine.UI.Image>().sprite = emeliaPortraitImage;
+        }
```

Comment:
the scriptable objects should allow for reduction of specific case handling here. Instead I would just add the images to the scriptable objects and assign from there based on the data.